gui/wxpython/location_wizard/wizard.py

```diff
@@ -2527,8 +2527,8 @@ def __init__(self, parent, grassdatabase):
                     parent=self.parent,
                     message="%s"
                     % _(
-                        "Unable to create new location. "
-                        "Location <%(loc)s> not created.\n\n"
+                        "Unable to create new project. "
+                        "Project <%(loc)s> not created.\n\n"
                         "Details: %(err)s"
                     )
                     % {"loc": self.startpage.location, "err": msg},
```

Comment:
The string still uses `(loc)` as the variable to replace. Are they side effects of adjusting the object used for interpolation?

Reply:
Not sure I understand, the intention here was just to replace user visible strings.

Reply:
Ok then, if you want to limit to strings only.